iphone/Classes/TiCalendarCalendar.m

```diff
@@ -175,8 +175,12 @@ -(NSArray*)getEventsInDate:(id)arg
     NSDateComponents *comps = [[NSDateComponents alloc] init];
     NSTimeInterval secondsPerDay = 24 * 60 * 60;
 
+    // We do this for parity with Javascript and Android
+    int month = [TiUtils intValue:[arg objectAtIndex:1]];
+    month += 1;
```

Comment:
Please explain this in case I miss something: When I supply 12 for December, it would use 13? I'd rather state the  valid range in the docs and validate that range before using it.